lib/jekyll/plugin_manager.rb

```diff
@@ -25,9 +25,7 @@ def conscientious_require
     #
     # Returns nothing.
     def require_gems
-      Jekyll::External.require_with_graceful_fail(
-        site.gems.select { |plugin| plugin_allowed?(plugin) }
-      )
+      Jekyll::External.require_with_graceful_fail(enabled_plugins)
     end
```

Comment:
I Just checked, same behavior as before. It throws a `Jekyll::Errors::MissingDependencyException` for that plugin.